### ISSUE TITLE
generated call graph has no edge attribute

### ISSUE THREAD

**Issue by ztjlime** [androguard/androguard#1034]
I'm trying to combine the call graphs generated by different sensitive api's into one graph
```python
H=nx.DiGraph()
for class_name in api_list:
     callgraph=dx.get_call_graph(classname=class_name,entry_points=entry_points)
     H=compose(H,callgraph)
nx.write_gml(H,out_path+'\\'+file_name+'.gml',stringizer=str)
```
However, when I specify the `classname` parameter in the `get_call_graph` function, the resulting `gml` file will only have the `node` attribute, not the `edge` attribute; when I don't specify the `classname`, there's the edge. 
Using the `androguard cg --classname`command in a terminal has the same problem

- Androguard Version: 4.10
- Python Version: 3.10
- Operating System: Windows 10

**Comment by ztjlime** [comment]
> What is the value of `class_name`? When I last fixed the callgraph generation feature (#985), I was able to run the following successfully with edges:
> 
> ```shell
> androguard --verbose cg --show --classname "Ltests/androguard/*" ~/projects/androguard/tests/data/APK/TestActivity.apk
> ```

The value of `class_name` is given by a list of sensitive apis containing 183 classes such as "Landroid/app/Service","Landroid/hardware/Camera", etc.
Also, to avoid the limitations of these classes, I use "Landroid/*" as the value of class_name, again without generating the edge attribute

**Comment by ztjlime** [comment]
I would like to extend my apologies for the issue I previously reported. The problem was with my dataset,and not with the code from your repository.  I found that only malware in it did not have the edge attribute, while benign software generated normally. I guess it's because the Android version of the malware is so old.
